metadata.graphqls

```diff
@@ -58,6 +58,11 @@ type Database {
     type: String!
 }
 
+type TimeInfo {
+  timezone: String
+  timestamp: Long
```

Comment:
Name as currentTime. And add coments about the format. What do you expect? Java timestamp?

Reply:
yes, Java timestamp.